dfget/core/downloader/p2p_downloader/p2p_downloader.go

```diff
@@ -240,7 +245,8 @@ func (p2p *P2PDownloader) pullPieceTask(item *Piece) (
 	if res == nil || (res.Code != constants.CodePeerContinue &&
```

Comment:
We should abstract this judgement, otherwise it is so unreadable for developers. 
Above comment is not related to this PR.